core/src/sql/bytes.rs

```diff
@@ -35,7 +34,15 @@ impl Deref for Bytes {
 
 impl Display for Bytes {
 	fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-		write!(f, "encoding::base64::decode(\"{}\")", STANDARD_NO_PAD.encode(&self.0))
+		writeln!(f, "")?;
+		let len = self.0.len();
+		if len > 0 {
+			for byte in &self.0[..len - 1] {
+				writeln!(f, "    {},", byte)?;
+			}
+			writeln!(f, "    {}", self.0[len - 1])?;
+		}
+		Ok(())
```

Comment:
This breaks imports and exports.

When exporting surrealql we current use the display trait, this is why bytes display is implemented returning a call to `encoding::base64::decode`, so that when displayed and outputted to an export, it will properly import again as a bytes array.

Reply:
Also this method makes it seem as if multiple values are returned, while it is only a single byte array value

Reply:
> This breaks imports and exports.
> 
> 
> 
> When exporting surrealql we current use the display trait, this is why bytes display is implemented returning a call to `encoding::base64::decode`, so that when displayed and outputted to an export, it will properly import again as a bytes array.

Shouldn't we dedicated export function then?

Reply:
@DelSkayn Do you suggest any changes or we will close this issue and PR

Reply:
> Also this method makes it seem as if multiple values are returned, while it is only a single byte array value

@DelSkayn I am printing byte array one by one there